docs/tasks.md

```diff
@@ -528,7 +528,7 @@ For example, `foo.Is-Bar_` is a valid parameter name for string or array type, b
 > 2. If a parameter name contains dots (.), it must be referenced by using the [bracket notation](#substituting-parameters-and-resources) with either single or double quotes i.e. `$(params['foo.bar'])`, `$(params["foo.bar"])`. See the following example for more information.
 
 #### Parameter type
-Each declared parameter has a `type` field, which can be set to `string`, `array` or `object` (alpha feature). 
+Each declared parameter has a `type` field, which can be set to `string`, `array` or `object` (beta feature).
```

Comment:
suggest labeling `array` with  `beta`  feature as well

```
Each declared parameter has a `type` field, which can be set to `string`, `array` (beta feature) or `object` (beta feature).
```

Reply:
yes that makes sense!